boards/mulle/include/periph_conf.h

```diff
@@ -265,9 +265,9 @@ static const spi_conf_t spi_config[] = {
         .pin_clk  = GPIO_PIN(PORT_D, 1),
         .pin_cs   = {
             GPIO_PIN(PORT_D, 0),
-            GPIO_UNDEF,
-            GPIO_UNDEF,
-            GPIO_UNDEF,
+            GPIO_PIN(PORT_D, 4),
+            GPIO_PIN(PORT_D, 5),
```

Comment:
Yes PTD5 is correct.
PTD0 is SPI0_PCS0, PTD4 is SPI0_PCS1, PTD5 is SPI0_PCS2. PTD6 is SPI0_PCS3